pkg/ansi/ansi.go

```diff
@@ -120,10 +120,10 @@ func getCharset() charset {
 
 const duration = time.Duration(100) * time.Millisecond
 
-// StartNewSpinner starts a new spinner with the given message. If the writer doesn't
-// support colors, it simply prints the message.
+// StartNewSpinner starts a new spinner with the given message. If the writer is not
+// a terminal or doesn't support colors, it simply prints the message.
 func StartNewSpinner(msg string, w io.Writer) *spinner.Spinner {
-	if !shouldUseColors(w) {
+	if !isTerminal(w) {
```

Comment:
Shouldn't this be the same as the corresponding check in StopSpinner()? 
```go
if !shouldUseColors(w) || !isTerminal(w) {
```

Reply:
Whoops, of course! Fixed now.